melos.yaml

```diff
@@ -45,7 +45,7 @@ scripts:
       - Requires running Android and iOS simulators.
 
   test:integration:android:
-    run: melos exec -c 1 "flutter drive --no-pub --driver=test_driver/integration_test.dart --target=integration_test/main_test.dart -d sdk"
+    run: melos exec -c 1 "flutter test --no-pub integration_test/main_test.dart -d sdk"
```

Comment:
not needed here, but at some point I think we can do away with all the main_test entry files and just point the command at the folder

Reply:
Definitely, yeah 👍